packages/components/src/avatar/docs/Avatar.stories.mdx

```diff
@@ -68,13 +68,11 @@ An avatar component can vary in size.
 <Preview scope={{ ChrisHadfield }}>
     <Story name="size">
         <Inline alignY="center">
-            <Avatar size="2xs" src={ChrisHadfield} name="Chris Hadfield" />
```

Comment:
Do we want to remove sizes? Are they no longer used?

Reply:
Since it's a fork of Orbit and will be deployed in its own scope on npm, nobody's using it yet.

Reply:
The 2xs is not used, bet the other ones are.